exercises/grains/test_grains.R

```diff
@@ -45,4 +45,15 @@ test_that("returns the total number of square on the board", {
   expect_equal(total(), 18446744073709551615)
 })
 
+# assist the "optimize for speed" instruction
+test_that("benchmarking", {
+  warning(
+    capture_output(
+      print = TRUE,
+      microbenchmark:::print.microbenchmark(
```

Comment:
You can remove this call to `print.microbenchmark` as it shouldn't be necessary.